statistics_templates.js

```diff
@@ -157,6 +157,43 @@ var getAllKinesisMetricsFuns = function(streamName) {
     ]
 }
 
+/**
+ * 
+ * @param {*} metricName - custom metric name 
+ * @param {*} collectorType - Collector type 
+ * @param {*} functionName - Lambda function name
+ * @param {*} namespace - Custom namespace 
+ * @param {*} StandardUnit - can be any value from this or string "Seconds"|"Microseconds"|"Milliseconds"|"Bytes"|"Kilobytes"|"Megabytes"|"Gigabytes"|"Terabytes"|"Bits"|"Kilobits"|"Megabits"|"Gigabits"|"Terabits"|"Percent"|"Count"|"Bytes/Second"|"Kilobytes/Second"|"Megabytes/Second"|"Gigabytes/Second"|"Terabytes/Second"|"Bits/Second"|"Kilobits/Second"|"Megabits/Second"|"Gigabits/Second"|"Terabits/Second"|"Count/Second"|"None"|string;
+ * @param {*} unitValue -value as per StandardUnit selected
+ * @param {*} callback 
+ * @returns 
+ */
+var reportCWMetric = function (metricName, collectorType, functionName, namespace, standardUnit, unitValue, callback) {
```

Comment:
please move this function to [`al_aws.js`](https://github.com/alertlogic/al-aws-collector-js/blob/master/al_aws.js)

Reply:
ok